README.md

```diff
@@ -40,6 +40,33 @@ options:
                         services.
   -e ENERGY, --energy ENERGY
                         Thermal energy. Between 0x0000 (light) and 0xffff (darker, default).
+  --no-resize           Disable automatic image resizing. Small images print as-is, 
+                        large images scale to width without changing height.
+```
+
+# New Features
+
+## Image Resize Control
+
+The `--no-resize` option provides more control over image printing:
+
+- By default, images are resized to match printer width
+- With `--no-resize`, small images print at original size
+- Large images are scaled to width while maintaining aspect ratio
+
+Example:
+
+```bash
+# Print without automatic resizing
+./print.py --no-resize test.png
+```
+
+## Printer Keep-Awake
+A keep-awake mechanism can be configured in the manager project to prevent printer sleep mode. This is typically managed externally and can be activated when running the server.
+
+```bash
+# Managed by parent project
+node server.js --keep-awake
```

Comment:
I suspect this is a leftover section that mentions "parent project". I would remove this from the PR.